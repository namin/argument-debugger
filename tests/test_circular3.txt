Free markets are the most efficient economic system.
We know this because free markets maximize economic efficiency.
Economic efficiency is best achieved through free market mechanisms.