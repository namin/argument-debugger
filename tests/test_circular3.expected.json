{
  "must_find": [],
  "may_find": [],
  "must_not_find": []
}